utils/game-utils.js

```diff
@@ -0,0 +1,29 @@
+const getLobbyDataByGame = (game) => {
```

Comment:
New utility to return a shared lobby object. I also added name to this so that when you reload the lobby we're able to persist that, too. Fully tested in the spec file.